git/gitconfig.txt

```diff
@@ -13,65 +13,92 @@
+    # Append staged changes to last commit without editing commit message.
     # Usage: git append
-    append = !git commit --amend --no-edit && :
+    append = "!f() { git commit --amend --no-edit; }; f"
+
+    # Shortens the given commit hash.
+    # Usage: git shorten <commit hash>
+    #        e.g. git shorten 1a2b3c4d5e6f7a8b9c0d1e2f3a4b5c6d7e8f9a0b
+    shorten = "!f() { local commitHash=$1 && git rev-parse --short $commitHash; }; f"
 
-    # Shortens the given commit hash
-    # Usage: git shorten [commit hash]
-    shorten = !git rev-parse --short $1 && :
+    # Change last commit's message.
+    # Usage: git reword <new message>
+    #        e.g. git reword foo
+    reword = "!f() { local newMessage=$1 && git commit --amend -m \"$newMessage\"; }; f"
 
-    # Change last commit's message
-    # Usage: git reword "[new message]"
-    reword = !git commit --amend -m \"$1\" && :
+    # Switches and pulls latest from given branch, prunes remote branches, fetches tags and deletes branches merged into given branch.
+    # Usage: git clean-merged [branch name (default: main)]
+    #        e.g. git clean-merged
+    #        e.g. git clean-merged feature/foo
+    clean-merged = "!f() { local targetBranch=${1:-main} && git switch $targetBranch && git pull -p --tags && git delete-merged $targetBranch; }; f"
 
-    # Switches and pulls latest from given branch, prunes remote branches, fetches tags and deletes branches squashed into given branch
-    # Usage: git cleanmerged [branch name]
-    #        e.g. git cleanmerged master
-    cleanmerged = !git switch $1 && git pull -p --tags && git delete-squashed $1 && :
+    # Switches and pulls latest from given branch, prunes remote branches, fetches tags and deletes branches squashed into given branch.
+    # Usage: git clean-squashed [branch name (default: main)]
+    #        e.g. git clean-squashed
+    #        e.g. git clean-squashed feature/foo
+    clean-squashed = "!f() { local targetBranch=${1:-main} && git switch $targetBranch && git pull -p --tags && git delete-squashed $targetBranch; }; f"
 
-    # Alias for cleanmerged
-    # Usage: git tidy [branch name]
-    tidy = !git cleanmerged $1
+    # Alias for clean-squashed.
+    # Usage: git tidy [branch name (default: main)]
+    #        e.g. git tidy
+    #        e.g. git tidy feature/foo
+    tidy = "!f() { local targetBranch=${1:-main} && git clean-squashed $targetBranch; }; f"
 
-    # Remove all dangling commits (including those still reachable from stashes and other reflogs)
+    # Remove all dangling commits (including those still reachable from stashes and other reflogs).
     # Based on https://stackoverflow.com/a/4528593
     # Usage: git spick-and-span
-    spick-and-span = !git reflog expire --expire-unreachable=now --all && git gc --prune=now && :
+    spick-and-span = "!f() { git reflog expire --expire-unreachable=now --all && git gc --prune=now; }; f"
```

Comment:
we could rename "spick-and-span" to "clean-repo" (or something similar) which seems more accurate and help to know what it will really do ;) - my bad when I created it 😅

Reply:
Done! I've opted for `purge-dangling`, as it clearly defines what the alias does.